antlr-kotlin-runtime/src/commonTest/kotlin/com/strumenta/minicalc/MiniCalcLexer.kt

```diff
@@ -23,12 +25,15 @@ class MiniCalcLexer(val input: CharStream) : Lexer(input) {
     override val atn: ATN
         get() = MiniCalcLexer.Companion.ATN
 
+    override val vocabulary: Vocabulary
+        get() = MiniCalcLexer.Companion.VOCABULARY
+
     @ThreadLocal
     companion object {
-        val decisionToDFA: Array<DFA>
+        val decisionToDFA : Array<DFA>
```

Comment:
this extra space seems out of place

Reply:
Yes, I agree, but it seems that this whitespace was added by @ftomassetti in Kotlin.stg :smiley:.
I just regenerated the file MiniCalcLexer.kt. Maybe we should add a way to do this automatically.

Reply:
Shame on this @ftomassetti ! What was he thinking :D ?